sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -24,82 +73,31 @@ class UniversalSpinTable(Base):
     repointing_number = Column(Integer, nullable=False)
 
 
-class FileCatalogTable:
-    """Common file catalog table"""
+class StatusTracking(Base):
+    """Status tracking table"""
+
+    __tablename__ = "status_tracking"
+
+    id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
+    file_to_create_path = Column(String, nullable=False)
+    status = Column(statuses, nullable=False)
+    job_definition = Column(String, nullable=False)
+    ingestion_date = Column(DateTime(timezone=True), nullable=True)
+
+
+class FileCatalog(Base):
+    """File catalog table"""
+
+    __tablename__ = "file_catalog"
 
     # TODO: determine cap for strings
     id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
     file_path = Column(String, nullable=False)
```

Comment:
Can be added in later, but did we want to have the "mission" qualifier as well, so we could have `imap-it`, `imap-dev` things like that to distinguish between dev/test/production data?

Reply:
Will we have that kind of data mixed together? I guess I was assuming we'd have separate accounts / deployments for each

Reply:
I think we should agree to keep them separate, either in different accounts or potentially in different tables. Mixing dev and production data together seems like not a good idea